common/util.c

```diff
@@ -71,6 +71,16 @@ float parse_float(const char *value) {
 	return flt;
 }
 
+bool parse_unit(const char *str, bool *ppt) {
```

Comment:
I don't think this needs to live in common code, since this is just used in `move.c`. A `static` function in the same file would be enough.